lib/slayer/form.rb

```diff
@@ -1,17 +1,31 @@
+require 'virtus'
+require 'dry-validation'
+
 module Slayer
   class Form
-    extend ModelAttribute
-    include ActiveModel::Validations if defined?(Rails)
+    attr_reader :errors
+    include Virtus.model
 
-    # Internal: Form Class Methods
     class << self
-      def from_params(params)
-        raise NotImplementedError
-      end
+      attr_accessor :_schema
 
-      def from_model(model)
-        raise NotImplementedError
+      def validations(&block)
```

Comment:
We might want to protect against double-calls to validations? e.g. what is the expected case if we run

```ruby
class PersonModel < Slayer::Form
  validations do
    # something
  end
  validations do
    # something else
  end
```

Does the second one always win?